### ISSUE TITLE
乱数の再現方法に関して

### ISSUE THREAD

**Issue by aisobe** [kikagaku/KIKAGAKU_Issues#24]
乱数の再現方法に関して
https://free.kikagaku.ai/tutorial/basic_of_computer_vision/learn/tensorflow_convolution_technic

畳み込みニューラルネットワークの精度向上
モデルの定義と学習
def reset_seed(seed=0):

最後の
from tensorflow.keras import models, layers
部分ですが、インデントされているように思います。これはreset_seedをcallする度に
import models, layers をやっていることになりますが、意図したコードでしょうか？

GPU環境での乱数の再現方法ができなくて困ってます。
Pytorchではできると聞いているのでtensorflowが原因だと思ってます。
